src/main/java/com/demo/MybatisApplication/dto/TeacherAdditionDto.java

```diff
@@ -0,0 +1,13 @@
+package com.demo.MybatisApplication.dto;
+
+import com.demo.MybatisApplication.model.SubjectEntity;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class TeacherAdditionDto {
```

Comment:
Why have added DTOs in this PR?
Is this because of merge/rebase error while resolving the conflicts?

Reply:
No, I was my fault. I did not add particular commit to add TeacherAddition and amended changes to this PR commit.